code/src/components/Loader.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled, { keyframes } from 'styled-components/macro'
+
+export const Loader = () => {
+  const isLoading = useSelector((state) => state.ui.isLoading)
+
+  return (
+    <>
+      {isLoading &&
+        <LoaderContainer>
+          <Loading>
+            <EmojiLoad>🥜</EmojiLoad>
+            <EmojiLoad>🥚</EmojiLoad>
+            <EmojiLoad>🥛</EmojiLoad>
+          </Loading>
+        </LoaderContainer>
+      }
+    </>
+  );
+};
+
+const LoaderContainer = styled.div`
```

Comment:
Your styled components should be defined above the actual react component. It works this way, but it's best practice to define things before they're used (reading from top to bottom)